src/Api.php

```diff
@@ -19,15 +19,14 @@ class Api
     // Our Slim app instance
     protected $app;
 
-    // Our Grav plugin endpoint settings
-    protected $settings;
-
     /**
-     * @param $config
+     * @var Config
      */
-    public function __construct($settings)
+    protected $settings;
+
+    public function __construct()
     {
-        $this->settings = Config::instance($settings);
+        $this->settings = Config::instance();
 
         // Initialise Slim
         $config = [
```

Comment:
Change this to `$slimConfig` and rename logger `name` field to the `grav-plugin-api`